frontend/src/scenes/surveys/SurveyCustomization.tsx

```diff
@@ -134,6 +135,43 @@ export function Customization({ appearance, surveyQuestionItem, onAppearanceChan
                         checked={appearance?.shuffleQuestions}
                     />
                 </div>
+                <div className="mt-1">
+                    <LemonField name="survey_popup_delay" className="font-medium">
+                        {({ onChange }) => {
+                            return (
+                                <div className="flex flex-row gap-2 items-center">
+                                    <LemonCheckbox
+                                        checked={!!appearance?.surveyPopupDelay}
+                                        onChange={(checked) => {
+                                            const surveyPopupDelay = checked ? 5 : undefined
+                                            onChange(surveyPopupDelay)
+                                            onAppearanceChange({ ...appearance, surveyPopupDelay })
+                                        }}
+                                    />
+                                    Delay survey popup after page load by{' '}
```

Comment:
The delay specified here is a minimum delay to show the popup, the actual delay depends on other surveys on the same page, etc.  Might be helpful to show that here. 

```suggestion
                                    Delay survey popup after page load by at least {' '}
```